pkg/kubelet/prober/prober.go

```diff
@@ -137,13 +144,16 @@ func (pb *prober) runProbe(ctx context.Context, probeType probeType, p *v1.Probe
 	timeout := time.Duration(p.TimeoutSeconds) * time.Second
 	switch {
 	case p.Exec != nil:
-		klog.V(4).InfoS("Exec-Probe runProbe", "pod", klog.KObj(pod), "containerName", container.Name, "execCommand", p.Exec.Command)
+		klog.V(3).InfoS("Exec-Probe runProbe", "pod", klog.KObj(pod), "containerName", container.Name, "execCommand", p.Exec.Command)
 		command := kubecontainer.ExpandContainerCommandOnlyStatic(p.Exec.Command, container.Env)
 		return pb.exec.Probe(pb.newExecInContainer(ctx, container, containerID, command, timeout))
 
 	case p.HTTPGet != nil:
 		req, err := httpprobe.NewRequestForHTTPGetAction(p.HTTPGet, &container, status.PodIP, "probe")
 		if err != nil {
+			// Log and record event for Unknown result
+			klog.V(3).InfoS("HTTP-Probe failed to create request", "error", err)
+			pb.recordContainerEvent(pod, &container, v1.EventTypeWarning, events.ContainerProbeWarning, "%s probe failed: %v", probeType, err)
```

Comment:
If you write an Event here, it will be recorded each time the probe goes to the Unknown state during a retry. But if the issue gets resolved through retries, I don’t think writing it as an Event is necessary. Just keeping it as a log message should be enough for debugging. If retries don’t fix the issue, it'll still be logged as an Event anyway. Same for l.173.